NEWS.md

```diff
@@ -1,6 +1,8 @@
 
 # rlang (development version)
 
+* `!!!` now uses a combination of `length()`, `names()`, and `[[` to splice
```

Comment:
This produces more consistent behaviour than `as.list()` on a wider variety of vector classes.

Reply:
And cross-link to the dplyr issue.